.github/actions/bot/index.js

```diff
@@ -49,17 +49,32 @@ async function bot(core, github, context, uuid) {
     }
 }
 
-// parseCommands splits the comment body into lines and parses each line as a command.
+// parseCommands splits the comment body into lines and parses each line as a command or named arguments to the previous command.
 function parseCommands(uuid, payload, commentBody) {
     const commands = [];
     if (!commentBody) {
         return commands;
     }
     const lines = commentBody.split(/\r?\n/);
     for (const line of lines) {
+        console.log(`Parsing line: ${line}`);
         const command = parseCommand(uuid, payload, line);
         if (command) {
             commands.push(command);
+        } else {
+            const namedArguments = parseNamedArguments(line);
+            if (namedArguments) {
+                const previousCommand = commands.at(-1);
+                if (previousCommand) {
+                    if (typeof previousCommand.addNamedArguments === 'function') {
+                        previousCommand.addNamedArguments(namedArguments.name, namedArguments.args);
+                    } else {
+                        console.log(`Parsed named arguments but previous command (${previousCommand.constructor.name}) does not support arguments: ${JSON.stringify(namedArguments)}`)
+                    }
+                } else {
+                    console.log(`Parsed named arguments with no previous command: ${JSON.stringify(namedArguments)}`)
+                }
+            }
```

Comment:
Should `previousCommand` be a required? can we validate the there is at least one line to reduce the nest if condition?

Reply:
I've been just logging and exiting the bot when things like this happen, but I'll add a function to reply to the user saying something went wrong 👍

Reply:
@Issacwww added in latest rev, see my fork for example usage: https://github.com/cartermckinnon/amazon-eks-ami/pull/6864#issuecomment-1758655539